etc/tmpl/edge.router.yml

```diff
@@ -81,6 +81,8 @@ dialers:
 
 listeners:
   - binding: tunnel
+    options:
+      mode: host
```

Comment:
i did a quick scan and did not see 'mode host' referenced. probably nbd but figured i'd point that out

Reply:
I added a note about it in os compatibility. It's also noted in the config section, which was in a previous PR